pillar/base/haproxy.sls

```diff
@@ -94,6 +94,11 @@ haproxy:
       verify_host: salt.psf.io
       check: "GET /.well-known/acme-challenge/sentinel HTTP/1.1\\r\\nHost:\\ salt.psf.io"
 
+    publish-files:
+      domains: []
+      verify_host: salt-public.psf.io
+      check: "GET /srv/public HTTP/1.1\\r\\nHost:\\ salt-public.psf.io"
```

Comment:
I'm not sure if this health check will pass based on how [try_files](https://nginx.org/en/docs/http/ngx_http_core_module.html#try_files) is configured.

Reply:
ah, yeah and the path is extraneous since nginx is configured to serve from /srv/public.

We could just have it `GET /salt-server-list.rst` for a check.